src/EmailReporter.ts

```diff
@@ -0,0 +1,66 @@
+import { readFileSync } from 'fs'
```

Comment:
This file should go under `reporters/email.ts`, and re-exported from `reporters/index.ts`.